### ISSUE TITLE
The training process is blocked early, with a very low loss and PSNR ~6.82

### ISSUE THREAD

**Issue by zhmen3** [xwx0924/SurgicalGaussian#3]
I used the default training parameters from `argument/endonerf/pulling.py` (cutting.py) and found that the loss was very small (<1) from the beginning, with PSNR consistently at a low level(5.17~6.82). The output renders **black** depth and **white** images.  
<img width="690" alt="Image" src="https://github.com/user-attachments/assets/1820536c-38fc-4bd3-bacf-a3f2c699669b" />

I checked the mask generated during the GS initialization phase, and it seems there are no issues. The problem might lie in unreasonable weight or loss design? can you assist in troubleshooting. 
<img width="945" alt="Image" src="https://github.com/user-attachments/assets/a196db5b-daa9-4955-9cb4-2ce6f7b1f857" />

If possible, pls provide a pretrained model for better paper reproduction.tks a lot:)

**Comment by xwx0924** [comment]
Sorry about that — the code was relatively simple at the time, so I recorded the test IDs manually. You can refer to the following configuration:
ModelParams = dict(
    dataset_type='endonerf',
    depth_scale=100.0,
    frame_nums=156,
    test_id=[1, 9, 17, 25, 33, 41, 49, 57, 65, 73, 81, 89, 97, 105, 113, 121, 129, 137, 145, 153],
    is_mask=True,
    depth_initial=True,
    accurate_mask=True,
    is_depth=True,
)
OptimizationParams = dict(
    iterations=40_000,
    warm_up=0,
    densify_until_iter=15_000,  
    lambda_smooth=0.005, 
    lambda_aiap_cov=100, 
    densify_grad_threshold=0.0003,
)
